spec/system/task_spec.rb

```diff
@@ -1,27 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  # == let!で前処理化==
+  let!(:task){ FactoryBot.create(:task) }
```

Comment:
省略できる定義は省略してしまいましょう！
```suggestion
  let!(:task){ create(:task) }
```